packages/ember-utils/lib/empty-object.js

```diff
@@ -1,8 +1,9 @@
-// This exists because `Object.create(null)` is absurdly slow compared
-// to `new EmptyObject()`. In either case, you want a null prototype
-// when you're treating the object instances as arbitrary dictionaries
-// and don't want your keys colliding with build-in methods on the
-// default object prototype.
+// This exists because `Object.create(null)` WAS slow compared to `new
```

Comment:
i thought ember-data used it, but it ships its own: https://github.com/emberjs/data/search?utf8=%E2%9C%93&q=EmptyObject&type=Code

Which we should then also remove